android/app/src/main/java/app/eduroam/geteduroam/profile/SelectProfileViewModel.kt

```diff
@@ -128,6 +138,37 @@ class SelectProfileViewModel @Inject constructor(
         }
     }
 
+    private fun loadDataFromCustomHost(context: Context) = viewModelScope.launch {
+        uiState = uiState.copy(inProgress = true)
+        try {
+            val profile = resolveLetswifiProfile(Profile(
+                id = customHost.toString(),
+                name = mapOf("any" to context.getString(R.string.name)),
+                type = Profile.Type.letswifi,
+                letswifiEndpoint = customHost.toString()
+            ))
+            uiState = uiState.copy(
+                inProgress = true,
```

Comment:
Yes, since this will always be a single-profile institution, it will immediately continue to the next step (OAuth screen), so it will still be loading